selene/src/roblox/base.toml

```diff
@@ -144,11 +157,7 @@ type = "number"
 required = false
 
 [[CFrame.new.args]]
-type = "any"
-required = false
-
-[[CFrame.new.args]]
-type = "any"
+type = { display = "Vector3" }
 required = false

```

Comment:
This declaration is wrong--it would error for `CFrame.new(0, 0, 0)` for example, and would also block `CFrame.new(Vector3.new(), Vector3.new())`.